targets/lenovo-x1/appvms/element.nix

```diff
@@ -46,9 +46,12 @@ in {
         };
       };
 
-      networking = pkgs.lib.mkIf isDendritePineconeEnabled {
-        firewall.allowedTCPPorts = [dendrite-pinecone.TcpPortInt];
-        firewall.allowedUDPPorts = [dendrite-pinecone.McastUdpPortInt];
+      networking = {
+        nameservers = ["192.168.100.1"];
```

Comment:
Can you please change this so that it comes from config? For example - `ghaf.networking.nameservers`.
There's parallel work ongoing by @tervis-unikie to bring `/etc/hosts` for static internal names and the current dnsmasq in the net-vm is to be replaced with external DNS only. Changing this would make the future adaptation easier. Thanks.